railties/lib/rails/generators/rails/plugin/templates/app/controllers/%namespaced_name%/application_controller.rb.tt

```diff
@@ -1,6 +1,5 @@
 <%= wrap_in_modules <<~rb
   class ApplicationController < ActionController::#{api? ? "API" : "Base"}
-    #{ api? ? '# ' : '' }protect_from_forgery with: :exception
```

Comment:
@rafaelfranca  I'm following the change made in https://github.com/rails/rails/pull/29742/files#diff-8380e0e11d4efeb2b0b142ae9e4cf512L2-L4 but do you recall why we do not add it to `ActionController::API`? `ActionController::API` doesn't inherit from `ActionController::Base` so there is some behavior changed introduced.

Reply:
`ActionController::API` doesn't support `protect_from_forgery`.

Reply:
To be a bit more clearer ... 😄 
`ActionController::Base` includes `RequestForgeryProtection`:
https://github.com/rails/rails/blob/03351cd5410b3ad3d3e51f44ce60aa553ddeef14/actionpack/lib/action_controller/base.rb#L227
`ActionController::API` does not:
https://github.com/rails/rails/blob/03351cd5410b3ad3d3e51f44ce60aa553ddeef14/actionpack/lib/action_controller/api.rb#L112

Reply:
Thank you for the explaination @p8 :) This means that this is actually a bug fix as well since including `protect_from_forgery`  when `--api` is passed to `rails plugin new` is incorrect.

Reply:
The line is commented out for API-only applications, so I wouldn't quite call it a bug fix, but not generating a comment with code that wouldn't work if it was uncommented is certainly a welcome change. 😄